sql/mob_groups.sql

```diff
@@ -12033,6 +12033,9 @@ INSERT INTO `mob_groups` VALUES (14335,5545,108,1800,0,4524,950,0,27,28,0);
 -- Chaneque
 INSERT INTO `mob_groups` VALUES (14336,5546,84,5400,0,4557,5000,0,56,57,0);
 
+-- Rumble Crawler that may spawn instead of Habetrot
+INSERT INTO `mob_groups` VALUES (14340,3415,159,1056,128,3015,0,0,53,56,0,0);
```

Comment:
please test before committing (you got an extra field in there so it wont import)